lib/phlex/elements.rb

```diff
@@ -20,11 +20,11 @@ def register_element(element, tag: element.name.tr("_", "-"))
 			def __phlex_#{element}__(**attributes, &block)
 				if attributes.length > 0
 					if block_given?
-						@_target << "<#{tag}" << (Phlex::ATTRIBUTE_CACHE[attributes.hash] || __attributes__(**attributes)) << ">"
+						@_target << "<#{tag}" << (Phlex::ATTRIBUTE_CACHE[process_attributes? ? (attributes.hash + self.class.hash) : attributes.hash] || __attributes__(**attributes)) << ">"
```

Comment:
It's significantly cheaper to do this cache check here rather than always calling `__attributes__` and having that method do the cache check.